staging/src/k8s.io/client-go/discovery/cached/disk/cached_discovery_test.go

```diff
@@ -36,72 +36,72 @@ import (
 )
 
 func TestCachedDiscoveryClient_Fresh(t *testing.T) {
-	assert := assert.New(t)
+	as := assert.New(t)
 
 	d, err := ioutil.TempDir("", "")
-	assert.NoError(err)
+	as.NoError(err)
 	defer os.RemoveAll(d)
 
 	c := fakeDiscoveryClient{}
 	cdc := newCachedDiscoveryClient(&c, d, 60*time.Second)
-	assert.True(cdc.Fresh(), "should be fresh after creation")
+	as.True(cdc.Fresh(), "should be fresh after creation")
 
 	cdc.ServerGroups()
-	assert.True(cdc.Fresh(), "should be fresh after groups call without cache")
-	assert.Equal(c.groupCalls, 1)
+	as.True(cdc.Fresh(), "should be fresh after groups call without cache")
+	as.Equal(c.groupCalls, 1)
 
 	cdc.ServerGroups()
-	assert.True(cdc.Fresh(), "should be fresh after another groups call")
-	assert.Equal(c.groupCalls, 1)
+	as.True(cdc.Fresh(), "should be fresh after another groups call")
+	as.Equal(c.groupCalls, 1)
 
-	cdc.ServerResources()
-	assert.True(cdc.Fresh(), "should be fresh after resources call")
-	assert.Equal(c.resourceCalls, 1)
+	cdc.ServerGroupsAndResources()
```

Comment:
Thanks for review @brianpursley. After careful consideration of your and @roycaihw's comments, these changes are not good fit for this PR. I have already opened this https://github.com/kubernetes/kubernetes/pull/107180 and moved these changes to there. Because `ServerResources` is deprecated function but out of scope of this PR.